pkg/kclient/kclient.go

```diff
@@ -200,6 +200,8 @@ func (c *Client) SetDiscoveryInterface(client discovery.DiscoveryInterface) {
 }
 
 func (c *Client) IsResourceSupported(apiGroup, apiVersion, resourceName string) (bool, error) {
+	klog.V(4).Infof("Checking is %s resource supported", resourceName)
```

Comment:
`Checking if %s resource is supported`

Also, I'd suggest `%q` instead of `%s`.